### PR TITLE
feat(dashboard-renderer): add metric card renderer [MA-2307]

### PR DESCRIPTION
**Pull request by adorack**
Currently just supports golden signals.  Styling TBD.

image

# Summary

#### Resources

- [Creating a package docs](https://github.com/Kong/public-ui-components/blob/main/docs/creating-a-package.md)

### CODE REVIEW

**Review by adamdehaven — APPROVED** [review]
lgtm

File: packages/analytics/dashboard-renderer/src/components/GoldenSignalsRenderer.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+const props = defineProps<RendererProps<MetricCardOptions>>()
+
+const options = computed<ProviderProps>(() => ({
+  longCardTitles: props.chartOptions.longCardTitles,
+  description: props.chartOptions.description,
+  hasTrendAccess: true,
+  refreshInterval: 0,
+}))
+</script>
```
Comment (filipgutica) [inline packages/analytics/dashboard-renderer/src/components/GoldenSignalsRenderer.vue:23]: We're not going to provide the flexibility yet for the host app to determine/define the metric cards they want to see, just keeping it at the three golden signals for now?

**Review by filipgutica — COMMENTED** [review]
Just had one comment

File: packages/analytics/dashboard-renderer/src/components/GoldenSignalsRenderer.vue (hunk 1 above)
Comment (adorack) [reply packages/analytics/dashboard-renderer/src/components/GoldenSignalsRenderer.vue:23]: That's correct.  A) There isn't any demand for that yet in any of the potential dashboards we want to create; and B) there's some complexity around which queries get issued, so it's not a feature we can just get for free.

**Pull request merged**